client/components/stars.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import propTypes from 'proptypes';
+import Star from './star';
+
+const Stars = (props) => {
```

Comment:
I'd consider destructuring within the arguments of this function like so: const Stars = ({ rating })